CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+## 1.1.0 (2021-04-29)
+
+
+### Features
+
+* added contrib rules ([ec60151](https://github.com/tobybatch/kimai2/commits/ec60151f86c7da710dd051d3f236e54cfcd2912a))
+* added linting to git commits ([3542601](https://github.com/tobybatch/kimai2/commits/3542601fd87d09a06c3b91ba32a3860f0e0550ef))
+* added nvmrc ([e3e53eb](https://github.com/tobybatch/kimai2/commits/e3e53eb3fc3db9239203da16d9fd811a840902aa))
+* addeed test script ([c24f649](https://github.com/tobybatch/kimai2/commits/c24f64981f09997e28d4dc026f132c7ba4dfbd44))
+
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+##  (2021-04-29)
+
+
+### Features
+
+* added contrib rules ([ec60151](https://github.com/tobybatch/kimai2/commits/ec60151f86c7da710dd051d3f236e54cfcd2912a))
+* added linting to git commits ([3542601](https://github.com/tobybatch/kimai2/commits/3542601fd87d09a06c3b91ba32a3860f0e0550ef))
+* added nvmrc ([e3e53eb](https://github.com/tobybatch/kimai2/commits/e3e53eb3fc3db9239203da16d9fd811a840902aa))
+* addeed test script ([c24f649](https://github.com/tobybatch/kimai2/commits/c24f64981f09997e28d4dc026f132c7ba4dfbd44))
```

Comment:
I think this needs to be removed?

Reply:
It's auto generated from the (old bad) commits. I'll try and figure out how to exclude it.